src/segmented-control/styles.scss

```diff
@@ -22,13 +25,28 @@
 .segment-part {
   display: -ms-inline-grid;
   display: inline-grid;
+
   @include styles.media-breakpoint-down(styles.$breakpoint-x-small) {
     display: none;
   }
+
+  &.refresh {
+    gap: 9px;
```

Comment:
magic numbers here again

Reply:
I updated and defined a root variable `$segment-divider-gap: awsui.$space-static-xxs;` in segment.scss that's used in relevant places.